src/components/InputErrorMessage/index.jsx

```diff
@@ -2,11 +2,15 @@ import React from "react";
 import "./index.scss";
 import patchExclamationFill from "../../images/icons/patchExclamationFill.svg";
 
-export default function InputErrorMessage({ message }) {
+export default function InputErrorMessage({ error }) {
+  if (!error) {
```

Comment:
shouldn't it check error.msg

Reply:
what did you mean? can you explain your point of view please?

This component will get the error object that related to each input validation. For example, I will render it for an email input like:
```
<InputErrorMessage error={errors.email} />
```
so if that error, related to email, not exists, the if part will handle it or the error message will be rendered and the message will be there for sure as the `errors.email` is there.

Reply:
is error.message guaranteed to be populated?

Reply:
yeah it must, so how the component would know if there is an error, as I should not be rendered in normal cases where there is not any validation error.

Reply:
i'm not sure i understand your answer but the safer thing to do here is to also add a check for error.message being undefined at the top. you shouldn't assume the fields are present